common/lib/protocol-definitions/src/test/validate0.1024.0.ts

```diff
@@ -0,0 +1,357 @@
+/*!
```

Comment:
is this file supposed to be an example of how the output should look?  should it stick around here if the tool is going to be integrated into the build script?